web/src/pages/logs-dev-page.tsx

```diff
@@ -140,7 +164,7 @@ const LogsDevPage: React.FunctionComponent = () => {
             <RefreshIntervalDropdown onRefresh={runQuery} />
             <Tooltip content={<div>Refresh</div>}>
               <Button
-                onClick={runQuery}
+                onClick={handleRefreshClick}
```

Comment:
```suggestion
                onClick={runQuery}
```

Reply:
The signatures are different so TypeScript does not like it

Reply:
👍